static/app/views/eventsV2/resultsChart.spec.tsx

```diff
@@ -130,5 +136,8 @@ describe('EventsV2 > ResultsChart', function () {
     expect(yAxisOptions.length).toEqual(2);
     expect(yAxisOptions[0].value).toEqual('count()');
     expect(yAxisOptions[1].value).toEqual('count_unique(user)');
+
+    // Wait for event geo request results to preopgate to update
+    await act(tick);
```

Comment:
at the end of the test? seems weird idk.

Reply:
yeah it's weird, idk either.

If we make a concerted effort to get rid of act and tick we can revisit then.

Reply:
I am down for this effort 😄